.github/workflows/nightly.yml

```diff
@@ -46,23 +46,13 @@ jobs:
           sudo add-apt-repository -y "deb [arch=arm64,s390x,ppc64el,riscv64] http://ports.ubuntu.com/ubuntu-ports/ $(lsb_release -sc) main" || true
```

Comment:
These lines should no longer be necessary either; the `crossbuild-essential-*` packages are "an {arm64,riscv,s390x} cross-compiler built on amd64" -- we have no need for foreign-arch libraries with this change.

Reply:
@neersighted removed the other arch repositories.